cmd/remove.go

```diff
@@ -86,5 +86,7 @@ func init() {
 	deleteCmd.PersistentFlags().StringVar(&raw.logVerbosity, "log-level", "INFO", "define the log verbosity for the log file, available levels: INFO(all requests/responses), WARNING(slow responses), ERROR(only failed requests), and NONE(no output logs).")
 	deleteCmd.PersistentFlags().StringVar(&raw.include, "include", "", "only include files whose name matches the pattern list. Example: *.jpg;*.pdf;exactName")
```

Comment:
renamed to include-name or include-pattern to be more specific?

Reply:
opted for include/exclude-pattern